setup.py

```diff
@@ -5,7 +5,7 @@
 import os
 import sys
 
-from setuptools import find_packages, setup
+from setuptools import setup
```

Comment:
It's probably a good idea to add some setuptools prerequisite like:
```suggestion
from setuptools import setup

__requires__ = ('setuptools >= 40', )
```

Reply:
added as `setup_requires` to `setup.cfg`

Reply:
`setup_requires` is different and it won't help because you cannot substitute already imported setuptools in runtime.

Reply:
`because you cannot substitute already imported setuptools in runtime.`